docs/code_ref/coordinates/carrington.rst

```diff
@@ -0,0 +1,126 @@
+.. _sunpy-coordinates-carrington:
+
+Calculating Carrington longitude
+================================
+
+Carrington coordinates are a heliographic coordinate system, where the longitude is determined assuming that the Sun has rotated at a sidereal period of ~25.38 days starting at zero longitude at a specific time on 1853 November 9.
+For a given observer at a given observation time, the center of the disk of the Sun as seen by that observer (also known as the apparent sub-observer point) has a Carrington latitude and longitude.
+When the observer is at Earth, the apparent sub-Earth Carrington latitude and longitude are also known as |B0| and |L0|, respectively.
+
+Quick summary
+-------------
+SunPy calculates the sub-observer Carrington longitude in a manner that enables co-alignment of images from different observatories at different locations/velocities in the solar system, but that results in a ~20-arcsecond discrepancy with |AA|.
+
+Observer effects
+----------------
+There are two primary effects for why the apparent sub-observer point is not the same as the "true" sub-observer point.
+
+Light travel time
+^^^^^^^^^^^^^^^^^
+
+It takes time for light to travel from the Sun to any observer (e.g., ~500 seconds for an observer at Earth).
+Thus, the farther the observer is from the Sun, the farther back in time the Sun will appear to be.
+This effect is sometimes called "planetary aberration".
+
+As an additional detail, the observer is closer to the sub-observer point than to the center of the Sun (i.e., the radius of the Sun), which corresponds to ~2.3 lightseconds.
```

Comment:
In the discussion above you said that  `HeliographicCarrington('earth', time)` is the Carrington frame at a time `time - light_travel_time_from_earth_to_sun_center`,  wouldn't be at sun surface?

Reply:
Ah, good point.  That earlier comment pre-dated some of the finer corrections implemented later.  I'll amend the earlier comment.